flytekit/core/base_task.py

```diff
@@ -901,3 +905,22 @@ def task_name(self, t: Task) -> Optional[str]:
         Overridable function that can optionally return a custom name for a given task
         """
         return None
+
+
+def decorate_python_task(task: PythonTask) -> Union[PythonTask, vscode]:
```

Comment:
can we inline this function? Its name gives the impression that it's a generic function, when in reality it decorates the function with the vscode decorator if the check passes.

Reply:
It's intentional. I'm assuming we will add Jupyter, PyCharm, or other decorators to Flytekit in the future.